Automattic/jetpack
# VideoPress: remove the video from UI

~branched off from https://github.com/Automattic/jetpack/pull/26405~

This PR implements the ability to remove the video from the UI connecting the components with the data layer.

Fixes https://github.com/Automattic/jetpack/issues/26402

#### Changes proposed in this Pull Request:

* VideoPress: remove the video from the UI.

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Go to dashboard
* Delete the video by clicking on the delete button fo the quick actions 
* Confirm the video is removed


https://user-images.githubusercontent.com/77539/192302601-23c5057f-fdb6-4ddb-9ad8-a067cc91400b.mov
